Proposal/KOHOUT_Project.cpp

```diff
@@ -0,0 +1,138 @@
+//Megan Kohout
+//final project
+#include <cstdlib>
+#include <iomanip>
+#include<iostream>
+#include <time.h>
+#include<fstream>
+
+using namespace std;
+
+ifstream gradeStorage ("/Users/kohou_000/Documents/School/19S/Intro to Programming/GitHub/cosc-a211-term-project-mekohout/Proposal/grade.txt");
+int responsePlaceHold=0;
+string className;
+int classNumber=0;
+int gradePlaceHold;
+int gradesAssignments[20];
+int numberAssignments;
+int weightAssignments;
+int averageAssignments;
+int weightedAssignments;
+//int gradesQuizzes[20];
+//numberQuizzes;
+//weightQuizzes;
+//int gradesTests[20];
+//numberTests;
+//weightTests;
+string userClassInput="class";
+int compositeGradeWeighted[10][3];//column: class row: 0=assignment weight, 1=quiz weight, 2=test weight
+
+void assignmentInformation (){
+  cout << "Does this class have assignments that are graded? If no, write 0. If yes, write how many."<< endl;
+  cin >> responsePlaceHold;//get their answer
+    if (responsePlaceHold==0){//if no asssignments then dont have var assignment
+      gradesAssignments[1]=0;
+      numberAssignments=0;
+    } else{//if there are assignments
+      numberAssignments=responsePlaceHold; //say how many things will be in the array
+      cout << "How much weight is given to assignments? For 70%, write .7."<<endl;
+      cin.ignore();
+      cin >> weightAssignments;
+
+      cout<< endl<<"rude";
+    }
+    cout << "hey";
+    } //this determines the assignment amount and how much the assignments are weighted. need to input actual grades still
+
+// void quizInformation(){
+//   cout << "Does this class have quizzes that are graded? If not, write no. If yes, write how many."<< endl;
+// }
+
+//void testInformation(){
```

Comment:
`testInformation()` is a sort of a noun—the name doesn't tell me what the function does.